NEWS

```diff
@@ -1,15 +1,15 @@
 #
-## Copyright (C) Mellanox Technologies Ltd. 2001-2019.  ALL RIGHTS RESERVED.
+## Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
 ## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
-## Copyright (C) ARM Ltd. 2017-2019.  ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
 
-## 1.7.0-rc1 (October 28, 2019)
+## 1.7.0-rc2 (January 16, 2019)
```

Comment:
2020?

Reply:
I didn't finish building my Time Machine, so has to be fixed